src/AppInstallerCommonCore/AppInstallerTelemetry.cpp

```diff
@@ -530,6 +530,19 @@ namespace AppInstaller::Logging
         }
     }
 
+    void TelemetryTraceLogger::LogNonFatalDOError(std::string_view url, HRESULT hr) const noexcept
+    {
+        if (IsTelemetryEnabled())
+        {
+            AICLI_TraceLoggingWriteActivity(
+                "NonFatalDOError",
+                TraceLoggingUInt32(s_subExecutionId, "SubExecutionId"),
+                AICLI_TraceLoggingStringView(url, "URL"),
```

Comment:
>URL [](http://example.com/codeflow?start=51&length=3)

Url is used as field name in above telemetry methods, we could probably save 1 small change in processing raw scripts by using same casing as previous